### DISCUSSION TITLE
How do I get this to work?

### DISCUSSION THREAD

**Discussion by Jareque** [matsn0w/IIS-EmergencyLS-ELS-FiveM#113] [Q&A]
I do think I installed everything correctly but the lights seem to not work.. Everything worked fine with mrdagrees ELS so is there a way anyone can help?

Discord - kryssy krys#0001

Thank you!

**Comment by matsn0w** [comment]
Hey, I'm sure we can find out what's wrong! For quick support, you can join my 'DM' Discord server [here](https://matsn0w.dev/discord).

Anyays, can you provide some more info about your configuration?
